tests/draft.test.js

```diff
@@ -0,0 +1,49 @@
+import '@babel/polyfill';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import status from '../src/helpers/constants/status.codes';
+import server from '../src/index';
+import signup from '../src/helpers/tests/signup';
+import errors from '../src/helpers/constants/error.messages';
+
+chai.use(chaiHttp);
+chai.should();
+
+const { OK, BAD_REQUEST } = status;
+const authToken = signup();
+let draftedSlug;
+describe('Drafting', () => {
+  it('should publish an article', (done) => {
+    chai
+      .request(server)
+      .patch('/api/articles/Test-draft-22khdb/publish')
+      .set('Authorization', authToken)
+      .end((err, res) => {
+        res.should.have.status(OK);
+        res.body.should.have.property('message',
+          'congratulations for making it, now you article is published');
+        done();
+      });
+  });
+
+  it('should not publish an article with missing property', (done) => {
```

Comment:
Is this the reason why you allowed null @alainburindi ?

Reply:
yes bro